plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/filtering/SemanticFilterRuleUtil.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ * 
+ * Copyright 2022 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package de.cau.cs.kieler.klighd.filtering;
+
+/**
+ * Util for easily getting some rules.
+ * 
+ * @author tik
+ *
+ */
+public abstract class SemanticFilterRuleUtil {
+    /** An empty tag used to define constants. */
+    private static final SemanticFilterTag EMPTY = new SemanticFilterTag("");
+    /** A rule that is always evaluated to false. */
+    public static final SemanticFilterRule FALSE =
+            new AndConnective(EMPTY, new NegationConnective(EMPTY));
+    /** A rule that is always evaluated to true. */
+    public static final SemanticFilterRule TRUE =
+            new OrConnective(EMPTY, new NegationConnective(EMPTY));
```

Comment:
These seem overly complicated. Why are they not static connectives without parameters that evaluate to true/false?